Snail/Snail+Combine/Observable+Combine.swift

```diff
@@ -0,0 +1,34 @@
+//  Copyright © 2021 Compass. All rights reserved.
+
+#if canImport(Combine)
+import Combine
+import Foundation
+
+@available(iOS 13.0, *)
+public extension ObservableType {
+    var publisher: AnyPublisher<T, Error> {
+        return SnailPublisher(upstream: self).eraseToAnyPublisher()
+    }
+
+    func asPublisher() -> AnyPublisher<T, Error> {
```

Comment:
what is the difference between this and the computed var?

Reply:
There's no difference. It's just a convenience to be able to access the underlying publisher either as `.asPublisher()` or `.publisher`

Reply:
Thoughts on naming this `asAnyPublisher()`?

This function returns an `AnyPublisher` and would make this more standard in my mind.

Reply:
Yeah we can totally rename it. While `.asPublisher()` might be shorter, `.asAnyPublisher()` is probably more descriptive.